### PR TITLE
Check for Windows Server version on prep testbed

### PR DESCRIPTION
**Pull request by jacekchalupka**

### CODE REVIEW

**Comment by jacekchalupka** [comment]
recheck no bug

File: ansible/roles/testbed/tasks/main.yml [hunk 1]
```diff
@@ -82,18 +82,25 @@
...
+  set_fact:
+    windows_server_version: "{{ ansible_distribution | regex_findall('Windows Server (\\d{4})') | first }}"
+
 # Workaround for flaky HNS behavior.
 # Removing container networks sometimes ends with "Unspecified error".
 # For environment cleanup in tests it is acceptable to just ignore this error
 # (-ErrorAction SilentlyContinue). However, this is not acceptable for
 # template creation (we don't want to have unreliable template).
-- name: Remove container networks (first attempt)
+- name: remove container networks on Windows Server 2016
```
Comment (michal-clapinski) [inline ansible/roles/testbed/tasks/main.yml]:
```suggestion
- name: Remove container networks on Windows Server 2016
```

File: ansible/roles/testbed/tasks/main.yml [hunk 2]
```diff
@@ -82,18 +82,25 @@
...
   win_shell: |
     Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
+    Get-ContainerNetwork | Remove-ContainerNetwork -Force
+  when: windows_server_version == '2016'
 
-- name: Remove container networks (final attempt)
+- name: remove container networks on Windows Server 2019
```
Comment (michal-clapinski) [inline ansible/roles/testbed/tasks/main.yml]:
```suggestion
- name: Remove container networks on Windows Server 2019
```

**Comment by jacekchalupka** [comment]
recheck no bug

**Comment by jacekchalupka** [comment]
recheck no bug

**Comment by jacekchalupka** [comment]
recheck no bug

**Pull request merged**